build.gradle

```diff
@@ -65,6 +65,10 @@ dependencies {
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: jUnitVersion
 }
 
+run {
```

Comment:
Assertions are already enabled in the master branch! Maybe you can merge master into your status branch first to avoid a merge conflict!